### DISCUSSION TITLE
More flexible gender options

### DISCUSSION THREAD

**Discussion by da5nsy** [Alovoa/alovoa#221] [Ideas]
I believe that good practice in collecting gender information is to allow write-ins, so that people who do not feel represented by 'man' or 'woman' can select something other than "other". Having to select "other" reinforces historical negative connotations of exclusion and deviance from the norm - a process known as "othering". I'll see if I can find some references to refer to.

I understand that this would introduce some technical issues, but I think it is worth considering if this is to be an inclusive project.

---

_Originally raised as #220_

**Comment by Nonononoki** [comment]
How about just renaming 'other' to 'non-binary'?

**Reply by da5nsy** [reply]
Unfortunately, that would leave lots of people feeling unrepresented.

**Comment by da5nsy** [comment]
Here's an article on the issue of gender in dating apps if anyone would like further info:

https://www.vice.com/en/article/z3v8bx/non-binary-people-dating-app-problems

**Comment by Leaced** [comment]
Non-binary is a community-chosen term and not othering a minority. In my opinion this would not be a solution, but it would be much better then it is now.

I think a second step for Alovoa could be to have 4 options:
- male
- female
- non-binary
- and a text field for writing any 4th option

The "looking for" section could then be:
- male
- female
- non-binary
With non-binary also including self-described genders. With this, we give the people a bit more possibilities, but we still use an umbrella term, which can not be chosen.

A Dating App with already very much possibilities about this is OkCupid. There it is possible to set multiple choices and also search for them. This could help us, creating something similar in the future.

When changing anything about this, #229 also needs to be worked on.
